homepage/homepage/content/docs/server-side/http-requests.mdx

```diff
@@ -200,28 +201,51 @@ export async function sendEventBookingRequest(event: Event) {
 
 ### Single Instance Requirements
 
-Deploy your Server Worker on a single instance when applying validation rules that require atomic operations.
+Some operations need to happen one at a time and in the same place, otherwise the data can get out of sync.
 
-For example, when implementing capacity limits in an event registration system, you need a single instance to ensure the capacity check and ticket creation happen atomically.
+For example, if you are checking capacity for an event and creating tickets, you must ensure only one server is doing it. 
+If multiple servers check at the same time, they might all think there is space and allow too many tickets.
 
-Jazz is an eventually consistent database, and with multiple region deployments you may accept multiple registrations at the same time
-and realize after the sync that the event is over capacity.
+Jazz uses eventual consistency (data takes a moment to sync between regions), so this problem is worse if you run multiple server copies in different locations.
 
-Transactional updates on multi-region deployments are part of the roadmap, but not yet supported.
+Until Jazz supports transactions across regions, the solution is to deploy a single server instance for these sensitive operations.
 
 Examples of when you must deploy on a single instance are:
-- when a counter is involved
-- when there is any kind of limit on the resources (like capacity)
-
-Essentially when the output of the request depends on the state of the database, you must deploy on a single instance.
+1. Distribute a limited number of tickets
+   * Limiting ticket sales so that only 100 tickets are sold for an event.
+   * The check (“is there space left?”) and ticket creation must happen together, or you risk overselling.
+2. Inventory stock deduction
+   * Managing a product stock count (e.g., 5 items left in store).
+   * Multiple instances could let multiple buyers purchase the last item at the same time.
+3. Rate limiting or quota tracking
+   * Enforcing a user can only perform 10 actions per day.
+   * Needs atomic updates to the counter to avoid bypassing the limit.
+4. Sequential ID or token generation
+   * Generating unique incremental order numbers (e.g., #1001, #1002).
+   * Multiple instances could produce duplicates if not coordinated.
+
+Single servers are necessary to enforce invariants or provide a consistent view of the data.
+
+As a simple rule to follow, when the output of the request depends on the state of the database, you should probably deploy on a single instance.
```

Comment:
_As a rule of thumb_, when the output...

(not sure why suggestions are not working, maybe the PR is too big?)